### DISCUSSION TITLE
Next Github Expert Application open

### DISCUSSION THREAD

**Discussion by Livinston-Bran** [community/community#66746] [Other Feature Feedback, Questions, & Ideas]

**Comment by fathi-riham-mn** [comment]
feb 1st week

**Comment by ghostinhershell** [comment]
Hey there 👋🏾 

GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community.

Furthermore, please note that the Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
